port_ocean/ocean.py

```diff
@@ -120,14 +125,15 @@ def initialize_app(self) -> None:
         async def lifecycle(_: FastAPI) -> AsyncIterator[None]:
             try:
                 await self.integration.start()
+                await self.webhook_manager.start_processing_event_messages()
                 await self._setup_scheduled_resync()
                 yield None
             except Exception:
                 logger.exception("Integration had a fatal error. Shutting down.")
                 logger.complete()
                 sys.exit("Server stopped")
             finally:
-                signal_handler.exit()
+                await signal_handler.exit()
```

Comment:
why 🤔

Reply:
Since the shutdown function on the processor_manager is async